controlplane/src/bin/seed-billing.ts

```diff
@@ -46,14 +48,17 @@ const seedBilling = async () => {
   const entries = Object.entries(json.plans);
 
   for (const [id, plan] of entries) {
-    const values = {
-      id,
+    const values: NewBillingPlan = {
+      id: id as FeatureIds,
```

Comment:
This type is incorrect, it's the id of the plan not a feature.

Reply:
Good catch